hooks/queries/votes/useAugmentedVoteData.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from "@tanstack/react-query";
+import { augmentedVoteDataKey } from "constant";
+import { useHandleError } from "hooks/helpers/useHandleError";
+import { getAugmentedVoteData } from "web3";
+import { useActiveVotes } from "./useActiveVotes";
+import { usePastVotes } from "./usePastVotes";
+import { useUpcomingVotes } from "./useUpcomingVotes";
+
+export function useAugmentedVoteData() {
```

Comment:
hook for getting augmented data. already sending all votes so that we don't have to update this in the mainnet task